src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const pop = document.querySelectorAll('.population');
+
+const text = [...pop].map(item => item.innerText.replace(/,/g, ''));
+
+const number = text.map(item => Number(item));
+
+const sum = number.reduce((item, a) => item + a, 0);
```

Comment:
```suggestion
const sum = number.reduce((sum, num) => sum + num, 0);
```

Reply:
use more meaningful names, your 'item' is actually accumulator that will be a result of .reduce
second parameter is current element of array

Reply:
Ok, thank you!